lib/hooks/usePantry.getVersions.ts

```diff
@@ -305,9 +305,7 @@ async function handleTransformer(transform: string, versions: string[]): Promise
     `));
   await writer.close()
 
-  const read = cmd.stdout.getReader().read().then(x => x.value)
-
-  const [out, { success: ok }] = await Promise.all([read, cmd.status])
+  const { stdout: out, success: ok } = await cmd.output()
```

Comment:
I think the docs imply you should await cmd.status but if it works maybe not then.

Reply:
Pulled that example out of the current docs, I believe. I'll add the reference.

Reply:
https://docs.deno.com/runtime/tutorials/subprocess#simple-example